backend/profiles/views.py

```diff
@@ -185,3 +187,115 @@ def badges(self, request, *args, **kwargs):
             }
 
         return Response(self.wrap_offer(data))
+
+    def wrap_follow_block(self, data, type, actor_name, object_name, summary_msg):
+        response = \
+            {
+                "@context": "https://www.w3.org/ns/activitystreams",
+                "summary": summary_msg,
+                "type": type,
+                "actor": {
+                    "type": "Person",
+                    "name": actor_name
+                },
+                "object":
+                    {
+                        "type": "Person",
+                        "identifier": object_name
+                    }
+            }
+        return response
+
+    @action(detail=True, methods=['post'], permission_classes=[permissions.IsAuthenticated])
+    def follow_user(self, request, *args, **kwargs):
+        user, _ = self.JWTauth.authenticate(self.request)
+        other_username = kwargs['pk']
+        other_user = self.queryset.get(username=other_username)
+        FollowRecord.objects.create(following_user=user.id, followed_user=other_user.id, follow_date=datetime.now())
```

Comment:
Seems like we don't need the `follow_date=datetime.now()` argument here, since it will be automatically set by Django if we give the argument `auto_now_add=True` to the model.